### ISSUE TITLE
Problems to render Maps after camera.rotateAround with certain points

### ISSUE THREAD

**Issue by LoveraSantiago** [libgdx/libgdx#4231]
#### Issue details
_ Sometimes the rotateAround method belonging to the Camera class is faulty in rendering OrthogonalTiledMapRenderer or TiledMapRenderer. I think it's a possible problem, otherwise I beg your pardon.

I came across the reported problem using the method in question in the game I'm trying to do. I thought it was something wrong with my code. Again went over my studies in Camera classes, OrthographicCamera and Viewport. Even redid the exercise of page

[https://github.com/libgdx/libgdx/wiki/Orthographic-camera](https://github.com/libgdx/libgdx/wiki/Orthographic-camera)

I made the inclusion of 2 more conditions within the handleInput method to rotate the screen with the rotateAround method and there were no problems with the rendering of Sprite.

But when the rotation occurs when using OrthogonalTiledMapRenderer or TiledMapRenderer rendering is weird depending on the point chosen for the rotateAround.

I posted a video. I used as an example to play the SuperKoalio. I ran two examples choosing 2 different positions to rotate the camera. In the right window the rotation takes place and the floor disappears. In the left window its just perfectly.
 I had the same problem in my game as the right window.

[https://www.youtube.com/watch?v=v35rHSwZhLE](https://www.youtube.com/watch?v=v35rHSwZhLE)

#### Reproduction steps/code
-To reproduce the error. I added the code line

`camera.rotateAround (new Vector3 (koala.position.x, koala.position.y, 0), new Vector3 (0,0,1), 1);`

after the updateKoala (deltaTime) within the render () method, after the line 131.
In the video example, the left screen that usually works the koala starting position on the line 116 remained the same.

`koala.position.set(20, 20);`

On the right screen the koala position was initiated with

`koala.position.set (30, 20);`

line 134 needs to be deleted to Koala be the center point.

#### Version of LibGDX and/or relevant dependencies
_Just tested with v9.3

#### Stacktrace
```java
//No stacktrace
```

#### Please select the affected platforms
- [x ] Android
- [?] iOS (robovm)
- [?] iOS (MOE)
- [?] HTML/GWT
- [ x] Windows
- [?] Linux
- [?] MacOS

Thank you. Libgdx is amazing!

**Comment by LoveraSantiago** [comment]
I edited, forgot to say that line 134 needs to be deleted to Koala be the center point.

**Comment by LoveraSantiago** [comment]
Hi Tom.

Thank you for your help.

I was really afraid that the issue that I mentioned was something wrong I did.
I saw that you fixed.


I would like to ask, how do I use this fix?

I have the libgdx source code in Eclipse I tried to hunt this error, but neither came close. Just take a git push and do rebuid with ant.

But how I use this fix in my game that uses the jars libgdx? I used libgdx setup to put together this little game.

Speaking of game I saw BlobRun and Somali. They are really beautiful, not yet played, but hope one day reach this level.
